packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.stories.js

```diff
@@ -13,7 +13,7 @@ import slashCommandSuggestion from './slash_command_suggestion';
 
 // Default Prop Values
 export const argsData = {
-  value: `I am not a standalone component, please use Message Input instead :v_tone3::robot:!`,
+  value: '<p>I am not a standalone component, please use Message Input instead<emoji-component code=":v_tone3:" id="270c-1f3fd"></emoji-component><emoji-component code=":robot:" id="1f916"></emoji-component> !</p>',
```

Comment:
Are those `id` attributes necessary?

Reply:
hmm it's definitely an attribute in the emoji extension but doesn't seem like it's used.. I'll look into it a bit more.

Reply:
Don't think it's necessary, removed.